src/helpers/getStatusBarText/index.ts

```diff
@@ -1,7 +1,15 @@
+import { workspace } from 'vscode'
+import { DEFAULT_TEXT } from '../../constants'
 import getFormattingStatus from '../getFormattingStatus'
 
-export const ENABLED_TEXT = 'Formatting: $(check)'
-export const DISABLED_TEXT = 'Formatting: $(x)'
+const configuration = workspace.getConfiguration()
+const statusBarText = configuration.get(
+  'formattingToggle.text',
+  DEFAULT_TEXT,
+)
```

Comment:
I don’t have much time to test this myself right now; but I assume right now you have to restart VS Code to have the new value get picked up? 

Super minor, but if you had this directly in `getStatusBarText()`, it would update the next time you click the status bar, no?

Reply:
Yes a restart is needed after the settings change. 

I'll move this around and get on-change settings to update automatically.

Reply:
Moved `statusBarLabel` and `configuration.get` into `getStatusBarText` — good call. This works automatically now!